book/src/installation_windows.md

```diff
@@ -29,13 +29,16 @@ When choosing the GTK version to build, select `gtk4` instead of `gtk3`:
 ```
 gvsbuild build gtk4
 ```
-
+<!--  -->
 ### Update `Path` environment variable
 
 2. Update your `Path` environment variable to include the GTK 4 libraries:
    1. Go to settings -> Search and open `Advanced system settings` -> Click on `Environment variables`
    2. Select `Path` -> Click on `Edit` -> Add `C:\gtk-build\gtk\x64\release\bin`
 
+3. Add `New User Variable` in environment variable to include `PKG_CONFIG_PATH`:
+   1. Go to settings -> Search and open `Advanced system settings` -> Click on `Environment variables`
+   3. Select `New` -> Input `Variable name` : `PKG_CONFIG_PATH` & `Variable value` : `C:\gtk-build\gtk\x64\release\lib\pkgconfig`
```

Comment:
Is this really necessary?
In the past iterations I didn't need this.

Reply:
While I was configuring, I encountered this requirement, and I was stuck on this problem for a while. Therefore, I believe it would be better to add it to the instructions so that beginners like me do not get stuck at this step in the future.